### PR TITLE
Feature/875

### PR DESCRIPTION
**Pull request by djhaskin987**
Fixes issue #875 by making sure the functions `install()` and `upgrade()` in the pre- and post-install scripts in RPM and similar functions in DEB packages don't explode.

### CODE REVIEW

**Comment by djhaskin987** [comment]
Did some cursory tests: Made the packages, ran the pre/post scripts involved to make sure they ran correctly :) :+1:

File: templates/deb/postinst_upgrade.sh.erb [hunk 1]
```diff
@@ -1,15 +1,21 @@
 #!/bin/sh
 after_upgrade() {
+<%# Making sure that at least one command is in the function -%>
+<%# avoids a lot of potential errors, including the case that -%>
+<%# the script is non-empty, but just whitespace and/or comments -%>
```
Comment (jordansissel) [inline templates/deb/postinst_upgrade.sh.erb]: +1 on your rationale

Comment (jordansissel) [inline templates/deb/postinst_upgrade.sh.erb]: My agreement, for posterity: We could also just sanitize ensure at least one character is non whitespace, but I think it's just safer to prepend `:` as the first command.

**Comment by jordansissel** [comment]
Eyeballed things; looks good to me. Didn't test, but it'll be allright :P

**Pull request merged**